scipy-notebook/Dockerfile

```diff
@@ -18,7 +18,7 @@ USER $NB_UID
 # Remove pyqt and qt pulled in for matplotlib since we're only ever going to
 # use notebook-friendly backends in these images
 RUN conda install --quiet --yes \
-    'blas=1.1*=openblas' \
+    'conda-forge::blas=*=openblas' \
```

Comment:
TIL. 👍

Reply:
FWIW [this line]( https://travis-ci.org/jupyter/docker-stacks/builds/387203458#L4819 ) shows that `conda` recognizes we have pinned the channel and build string of this package in the environment.

Reply:
Should add details about MatchSpec are in the [4.4.0 release notes]( https://github.com/conda/conda/releases/tag/4.4.0 ).